.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "googletest"]
```

Comment:
Why use a whole submodule for two tests? [Catch2](https://github.com/catchorg/Catch2) would be a better fit to stay small.